test/remote/gateways/remote_trust_commerce_test.rb

```diff
@@ -182,6 +182,13 @@ def test_failed_store
     assert_bad_data_response(response)
   end
 
+  def test_successful_unstore
+    assert store = @gateway.store(@credit_card)
+    assert_equal 'approved', store.params['status']
+    assert response = @gateway.unstore(store.params['billingid'])
+    assert_success response
+  end
+
```

Comment:
@jherreraa just to have a better remote tests and to ensure that store works as expected, please add a test that can use an stored payment method to do a purchase/authorize

Reply:
Test added! Please check it again!